daemon/seccomp_disabled.go

```diff
@@ -0,0 +1,12 @@
+// +build !seccomp,!windows
```

Comment:
OK, it looks like that commit was after my last update, I have updated my local branch and this is no longer an issue. Thank you. I'll delete my comments.